packages/paste-core/components/avatar/stories/index.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Stack} from '@twilio-paste/stack';
+import {Avatar, AvatarImg} from '../src';
+
+storiesOf('Components|Avatar', module)
+  .addDecorator(withKnobs)
+  .add('Initials', () => {
+    return (
+      <Stack orientation="horizontal" spacing="space40">
+        <Avatar size="sizeIcon10">ST</Avatar>
+        <Avatar size="sizeIcon20">ST</Avatar>
+        <Avatar size="sizeIcon30">ST</Avatar>
+        <Avatar size="sizeIcon40">ST</Avatar>
+        <Avatar size="sizeIcon50">ST</Avatar>
+        <Avatar size="sizeIcon60">ST</Avatar>
+        <Avatar size="sizeIcon70">ST</Avatar>
+        <Avatar size="sizeIcon80">ST</Avatar>
+        <Avatar size="sizeIcon90">ST</Avatar>
+        <Avatar size="sizeIcon100">ST</Avatar>
+        <Avatar size="sizeIcon110">ST</Avatar>
+      </Stack>
+    );
+  })
+  .add('Image', () => {
+    return (
+      <Stack orientation="horizontal" spacing="space40">
+        <Avatar size="sizeIcon10">
+          <AvatarImg alt="avatar example" src="/avatars/avatar2.png" />
```

Comment:
Looking at this usage code, I'm starting to wonder what value having a composable `AvatarImg` provides 🤔 , maybe we just these as props to simplify the whole thing, and make `children?: string`

Reply:
Just pass `alt` and `src` to avatar if you want an image?

Reply:
Going with this, tidies things up a bunch